### PR TITLE
fix(typos): Fix minor typos in the source code

### PR DESCRIPTION
**Pull request by mribeirodantas**
## :bookmark_tabs: Summary

Fix minor typos in the source code and other files

Make sure you

- [X] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [NA] :computer: have added necessary unit/e2e tests.
- [NA] :notebook: have added documentation. Make sure [`MERMAID_RELEASE_VERSION`](https://github.com/mermaid-js/mermaid/blob/develop/packages/mermaid/src/docs/community/development.md#3-update-documentation) is used for all new features.
- [NA] :bookmark: targeted `develop` branch

### CODE REVIEW

**Comment by mribeirodantas** [comment]
When I try to fix linting with `prettier`, it seems to fix much more than the files I changed. What should I do?

File: packages/mermaid/src/diagrams/sequence/sequenceDb.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ export const addSignal = function (
     const cnt = activationCount(idFrom.actor);
     if (cnt < 1) {
       // Bail out as there is an activation signal from an inactive participant
-      let error = new Error('Trying to inactivate an inactive participant (' + idFrom.actor + ')');
+      let error = new Error('Trying to deactivate an inactive participant (' + idFrom.actor + ')');
```
Comment (sidharthv96) [inline packages/mermaid/src/diagrams/sequence/sequenceDb.js:129]: This should be reverted, as it is a breaking change. We are changing error text, that's sent outside.

**Review by sidharthv96 — APPROVED** [review]
Nice work!

**Comment by sidharthv96** [comment]
The unit test is failing because the error text has changed. That should be reverted. 
You can create another PR after this one is merged, targeting `next` branch, where we can have breaking changes. 

re Prettier, please run the command mentioned in the error, commit and push. Currently the change in `packages/mermaid/src/docs/syntax/quadrantChart.md` is to fix spacing of the `|` of the table.

File: packages/mermaid/src/diagrams/sequence/sequenceDb.js (hunk 1 above)
Comment (mribeirodantas) [reply packages/mermaid/src/diagrams/sequence/sequenceDb.js]: Reverted. Thanks for the feedback, @sidharthv96 ! 🥳

**Comment by mribeirodantas** [comment]
Thanks for the feedback, @sidharthv96! LGTM? 😃

**Pull request merged**